reference/request-bodies/payment_method_request.yaml

```
@@ -94,6 +95,24 @@ properties:
       require buyer approval.
     example: https://example.com/callback
 
+  currency:
+    type: string
+    example: USD
+    description: |-
+      The ISO-4217 currency code to store this payment method for. This is
+      used to select the payment service to use.
+
+      This only applies to `redirect` payment methods like `gocardless`.
```

```suggestion
      This only applies to `redirect` mode payment methods like `gocardless`.
```

make it clear that `redirect` applies to the `mode` of the payment method.